python/Makefile

```diff
@@ -0,0 +1,122 @@
+#
+# This Makefile serves as convenient command-line auto-completion when working on Python bindings
+#
+TOOLBOX_DIR?=../toolbox
+PROJECT_NAME=xnvme
+XNVME_CAPI_HEADERS=$(shell ls ../include/libxnvme* | grep -v util | grep -v libxnvmec)
+
+define default-help
+# invoke: 'make uninstall', 'make install'
+endef
+.PHONY: default
+default: uninstall install
```

Comment:
I think we should make the default `generate-ctypes patch-ctypes`, as I understand you'd idiomatically write `make && make install`.